src/internal/kindenv/kindenv.go

```diff
@@ -158,8 +158,11 @@ type ClusterConfig struct {
 }
 
 func (c *Cluster) GetConfig(ctx context.Context, namespace string) (*ClusterConfig, error) {
-	if c.config != nil {
-		return c.config, nil
+	if c.config == nil {
+		c.config = make(map[string]*ClusterConfig)
+	}
```

Comment:
This _could_ be an `else`, but I don’t know if that makes it more or less readable.